types/wordpress__components/package.json

```diff
@@ -2,7 +2,7 @@
     "private": true,
     "dependencies": {
         "@wordpress/element": "^3.0.0",
-        "downshift": "^6.0.15",
+        "downshift": "6.0.15",
```

Comment:
Do we need this change here?

Reply:
Actually `downshift`'s latest version is causing build failures. [Here are the logs of the CI run.](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/938f33799d3b79ebec3bd5d4dba4c2c258096b2d/checks?check_suite_id=3408195176)

Reply:
Alright, then let's revert it here and wait for `master` to get fixed.